rados/rados.go

```diff
@@ -96,17 +107,28 @@ func (c *RadosConn) MonCommand(args []byte) (buffer []byte, info string, err err
 
 	conn, err := c.newRadosConn()
 	if err != nil {
-		return nil, "", err
+		return
 	}
-	defer conn.Shutdown()
 
+	defer conn.Shutdown()
 	ll = ll.WithField("conn", conn.GetInstanceID())
 
-	ll.Trace("start executing mon command")
-
-	buffer, info, err = conn.MonCommand(args)
-
-	ll.WithError(err).Trace("complete executing mon command")
+	ch := make(chan bool, 1)
+	go func() {
+		ll.Trace("start executing mon command")
+		buffer, info, err = conn.MonCommand(args)
+		ch <- true
+	}()
```

Comment:
Same comments here as above.

Reply:
-resolving due to earlier code change- 🙂